dev/com.ibm.ws.org.hibernate.validator.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/LibertyValidatorFactoryBean.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2015 IBM Corporation and others.
```

Comment:
can undo changes to this file

Reply:
Change due to basing off of feature branch and then switching to master.